py-rse-index.Rmd

```diff
@@ -22,25 +22,78 @@ source(here::here("_common.R"))
 >
 > — Terry Pratchett
 
-Computers are now as essential for research as telescopes, test tubes, and libraries,
-which means that researchers need need to know how to build, use, and share software.
+Software is now as essential to research as telescopes, test tubes, and reference libraries,
+which means that researchers need need to know how to build, check, use, and share programs.
 However,
 most introductions to programming focus on developing commercial applications,
-not on exploring problems and answering questions.
-This series of books will show you how to do the latter
+not on exploring problems whose answers aren't yet known.
+Our goal is show you how to do that,
 both on your own and as part of a team.
 
 The prequels to this book introduce three core skills:
 how to write short programs to clean and analyze data in a reproducible way,
```

Comment:
If this gets published first as a stand-alone book there are no prequels....

Reply:
Good catch - fixed.